EIPS/eip-draft_restrict_precompile_address_range.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 1352
+title: Specify restricted address range for precompiles/system contracts
+author: Alex Beregszaszi (axic)
+discussions-to: https://ethereum-magicians.org/t/eip-1352-specify-restricted-address-range-for-precompiles-system-contracts/1151
+status: Draft
+type: Standards
+category: Core
+created: 2018-07-27
+---
+
+## Simple Summary
+Specify an Ethereum address range occupied by precompiles and future system contracts. Regular accounts and contracts cannot obtain such an address.
+
+## Abstract
+The address range between 0x0000000000000000000000000000000000000000 and 0x00000000000000000000000000000000000000ff is reserved for precompiles and system contracts.
+
+## Motivation
+This will simplify certain future features where unless this is implemented, several exceptions must be specified.
+
+## Specification
+The address range between 0x0000000000000000000000000000000000000000 and 0x00000000000000000000000000000000000000ff is reserved for precompiles and system contracts.
+
+If a contract creation (as a result of a create transaction or a `CREATE` opcode) results in an address within this range, then it is rejected.
```

Comment:
Is this even worth checking for?

Reply:
If it is not specified then clients will introduce random assumptions. Why is it better not to specify it even if probability of extremely low?

Reply:
One reason against adding that clause is that there doesn't seem to be an easy way to add a test for it without changing internals :)

Reply:
Please clarify: is this a comment blocking the merging of this draft or should this properly be evaluated on the discussion url?

I could also see a middle ground by stating: due to the low probability (and lack of adequate testing possibilities) no explicit checks should be added to ensure that external transaction signing or the invoking of the `CREATE` instruction can result in a precompile address.